packages/nimble-components/src/table/template.ts

```diff
@@ -47,14 +51,24 @@ export const template = html<Table>`
                             </${checkboxTag}>
                         </span>
                     `)}
-
+                    <span role="gridcell">
+                        <${buttonTag}
+                            class="${x => `collapse-all-button ${(x.showCollapseAll ? 'visible' : '')}`}"
```

Comment:
You can simplify this by keeping the fixed class name outside of the template insertion point

```suggestion
                            class="collapse-all-button ${x => x.showCollapseAll ? 'visible' : ''}"
```

Should be able to split on multiple lines if many conditional classes.

```ts
class="
    collapse-all-button
    ${x => x.showCollapseAll ? 'visible' : ''}
"
```